releases/v1.7.0-beta.toml

```diff
@@ -0,0 +1,95 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.6.0"
+
+pre_release = true
+
+preface = """\
+The eighth major release of containerd includes new functionality alongside many improvements.
+This release is intended to be the last major release of containerd 1.x before 2.0.
+Some functionality in this release may be considered experimental or unstable, but will become stable or default in 2.0.
```

Comment:
We need the complete list of the experimental features, but it can be deferred until v1.7 GA
https://github.com/containerd/containerd/issues/7020

Reply:
agreed